app/models/concerns/courses/edit_options.rb

```diff
@@ -26,6 +26,7 @@ def edit_course_options
           show_start_date: show_start_date?,
           show_applications_open: show_applications_open?,
           subjects: potential_subjects,
+          potential_modern_languages: potential_modern_languages,
```

Comment:
I meant here

Reply:
Same as my previous comment

Reply:
i disagree...you either have it or not have but not potential having it not having it

Reply:
These are intended to be the modern languages a user may be able to select from if any.

Reply:
I think I agree about making the naming convention standard, and removing `potential` from the key, here. I would go one step further and favour the language `available` over `potential`, but that's less important.

```suggestion
          modern_languages: available_modern_languages,
```